.github/workflows/build-push-images.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and Push Images
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
maybe we want to add release-* ?

We can also do that at the release branch itself.

Reply:
does wildcard work on branch names?

Reply:
ok, it [does](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags)